src/compose/application-manager.ts

```diff
@@ -680,9 +680,9 @@ function saveAndRemoveImages(
 				}) ?? _.find(availableImages, { dockerImageId: svc.config.image }),
 		),
 	) as imageManager.Image[];
-	const targetImages = _.flatMap(target, (app) =>
-		_.map(app.services, imageForService),
-	);
+
+	const targetServices = Object.values(target).flatMap((app) => app.services);
```

Comment:
this is unfortunate we have to do Object.values but i'm sure lodash was doing exactly this.

Reply:
That's probably right. There is no other option as target is an object